react/features/participants-pane/components/web/MeetingParticipants.tsx

```diff
@@ -34,17 +37,17 @@ import MeetingParticipantContextMenu from './MeetingParticipantContextMenu';
 // @ts-ignore
 import MeetingParticipantItems from './MeetingParticipantItems';
 
-const useStyles = makeStyles((theme: any) => {
+const useStyles = makeStyles()((theme: Theme) => {
     return {
         heading: {
             color: theme.palette.text02,
-            ...theme.typography.labelButton,
-            lineHeight: `${theme.typography.labelButton.lineHeight}px`,
+            // @ts-ignore
+            ...withPixelLineHeight(theme.typography.labelButton),
             margin: `8px 0 ${participantsPaneTheme.panePadding}px`,
 
             [`@media(max-width: ${participantsPaneTheme.MD_BREAKPOINT})`]: {
-                ...theme.typography.labelButtonLarge,
-                lineHeight: `${theme.typography.labelButtonLarge.lineHeight}px`
+                // @ts-ignore
+                ...withPixelLineHeight(theme.typography.labelButtonLarge)
```

Comment:
TS does not like the spread operator? Or it does not assert that the returned value of  `withPixelLineHeight` is an object?

Reply:
The `labelButtonLarge` is a token that is not part of the Design System, therefore it was also not added to the Theme interface. The plan is to change its use with something else